mbed_lib.json

```diff
@@ -2,7 +2,9 @@
     "name": "easy-connect",
     "target_overrides": {
         "*": {
-            "target.features_add": ["COMMON_PAL"]
+            "target.features_add": ["COMMON_PAL"],
+            "platform.stdio-baud-rate":115200,
```

Comment:
I don't want this in mbed_lib.json. This should be an option for the application. Not a library.